### ISSUE TITLE
Issue with 2 instances of Jitsi Meet and self hosting Openfire running into 2 different domains

### ISSUE THREAD

**Issue by ErwanF76** [jitsi/jitsi-meet#7405]
Hi

I'm facing some issues to make 

A short summary of my settings:

2 Openfire self hosting servers running into 2 differents domains;
The ofmeeting plug-in embedding Jitsi Meet is installed and running at each Openfire instances

That gives:
Client 1 (Edge Chromium) <-> Openfire@domain1.org <-> Openfire@domain2.org <-> Client 2 (Edge Chromium)
Everything works fine (videoconf between the 2 domains) if I use Openfire Pàdé.

But using the webclient of Jitsi Meet in Edge Chromium, I don't get how to make both sides using the same room:
If I typed TESTROOM at client 1, TESTROOM session is created in domain domain1.org => that creates a new chatroom TESTROOM@domain1.org.
If I typed TESTROOM at client 2, TESTROOM session is created in domain domain2.org => that creates a new chatroom TESTROOM@domain2.org.
So basically, the 2 end users don't join the same room and can't communicate with each other.
Note that it works with Pàdé extension since domains are well specified.

So I wonder how Jitsi Meet manages communication involving clients and video conference rooms which are not in the same DNS domain ? Is Jitsi conference focus supposed to manage this ? I'm a bit lost :(

Many thx for the clarification !
KR
ErwanF

**Comment by Echolon** [comment]
Hi,
If you have a question about Jitsi Meet that is not a bug report or feature request, please post it in https://community.jitsi.org

Unfortunately, I have to close this :(

---
*I'm a community contributor and not directly affiliated with the Jitsi team.*

**Comment by ErwanF76** [comment]
Ok, sorry. My mistake.Kind regards
Envoyé depuis Yahoo Mail pour Android 
 
  Le lun., juil. 27, 2020 à 19:45, Echolon<notifications@github.com> a écrit :   


Hi,
If you have a question about Jitsi Meet that is not a bug report or feature request, please post it in https://community.jitsi.org

Unfortunately, I have to close this :(

I'm a community contributor and not directly affiliated with the Jitsi team.
